.eslintrc.json

```diff
@@ -117,6 +117,12 @@
                 "@typescript-eslint/func-call-spacing": ["error"],
                 "@typescript-eslint/keyword-spacing": ["error"]
             }
+        },
+        {
+            "files": ["vitest.config.ts"],
```

Comment:
This file doesn't exist (and perhaps should remain the .js file you currently include)